### DISCUSSION TITLE
Question regarding organization naming

### DISCUSSION THREAD

**Discussion by ZaLiTHkA** [community/community#145090] [npm] [answered]
I'm having issues trying to register an organization on the NPM registry... checking inside DevTools, I see the HTTP request to `https://www.npmjs.com/org/create?track=orgUpgradeSuccess` returns a 404 response when I click "Create", but the web UI is giving me nothing.

when I try navigate to the example URL shown above the "Name" field in the "Create New Organization" form, it takes me to a user's page with the URL `npmjs.com/~<name>` (which is not my current user), whereas when I navigate to an *actual* organization's page, I see the URL `npmjs.com/org/<name>`.

are we not able to create an organization that is exactly the same as an existing username? or is there perhaps something else going wrong here?

**Comment by monishcm** [comment, marked as answer]
You can convert your username to an Org account. You cannot have a personal account and an Org account with the same name.

**Reply by ZaLiTHkA** [reply]
gotcha, thanks for the clarification.
